specs/agents/tracing-instrumentation-db.md

```diff
@@ -44,13 +44,13 @@ The following fields are relevant for database and datastore spans. Where possib
 
 | Field | Value / Examples | Comments |
 |-------|:---------------:|----------|
-|`name`| e.g. `DynamoDB UpdateItem my_table`|  The span name should capture the operation name (as used by AWS for the action name) and the table name, if available. The format should be `DynamoDB <ActionName> <TableName>`. |
+|`name`| e.g. `DynamoDB UpdateItem my_table`|  The span name should capture the operation name (as used by AWS for the action name) and the table name, if available. The format should be `DynamoDB <ActionName> <TableName>`. TableName MAY be omitted from the name for operations (`batchWriteItem`, `batchGetItem`, PartiQL-related methods like `executeStatement` etc.) that are acting on more than a single table. If `TableName` is not available, agents SHOULD also check the `TableArn` or `SourceTableArn` query params for a table name and extract the table name from the AWS ARN value.|
 |`type`|`db`|
 |`subtype`|`dynamodb`|
 |`action`| `query` |
 | __**context.db._**__ |<hr/>|<hr/>|
 |`_.instance`| e.g. `us-east-1` | The AWS region where the table is. |
-|`_.statement`| e.g. `ForumName = :name and Subject = :sub` | For a DynamoDB Query operation, capture the KeyConditionExpression in this field. |
+|`_.statement`| e.g. `ForumName = :name and Subject = :sub` | For a DynamoDB Query operation, capture the KeyConditionExpression in this field. In order to avoid a high cardinality of collected values, agents SHOULD NOT include the full SQL statment for PartiQL-related methods like `executeStatment.|
```

Comment:
`span.context.db.statement` is *not* a field that is concerned with cardinality, though, right?   So I'd think the PartiQL query should be fine here, just as the full SQL statement or ES query for those spans.

Reply:
@astorm Ping. I don't recall why this question was marked resolved.

Reply:
It is safe to say that `db.statement` is not on the "concerned with cardinality" list because the spec for the `db.statement` for other databases states to store the full statement.

> The full database statement should be stored in db.statement, which may be useful for debugging performance issues. We store up to 10000 Unicode characters per database statement. 

> The MongoDB command encoded as MongoDB Extended JSON.

> For Elasticsearch search-type queries, the request body may be recorded.

I don't think DynamoDB should be any different.

Reply:
@trentm sounds good Trent -- you should have rights to branch to make any changes you'd like.